app/assets/javascripts/app/app_router.coffee

```diff
@@ -0,0 +1,21 @@
+Router = Backbone.Router.extend {
+  routes:
+    ':name'   : 'show'
+
+  show: (name) ->
+    @showPage(name)
+
+  showPage: (name) ->
+    window.peopleList = new PeopleList
+    peopleList.fetch().then ->
+      $('.card').remove()
+      $.each(peopleList._byId, (_prop, person_data) ->
+        slug =  person_data.attributes._url.split('/')[2].split('.')[0]
+        $('.backbone').html person_data.attributes.bio if name == slug
```

Comment:
Coffeescript provides things like `is` , `isnt` for comparisons.

e.g.  `if name is slug` which actually uses the more correct `===` equality check. See example here: http://coffeescript.org/#try:foo()%20if%20name%20is%20slug